src/AddTodoForm.jsx

```diff
@@ -1,10 +1,17 @@
 import React from "react";
 
-function AddTodoForm() {
+function AddTodoForm(props) {
+  const handleAddTodo = (event) => {
+    event.preventDefault(); // Prevent default form submission behavior
+    const todoTitle = event.target.title.value; // Get value of the "title" input
```

Comment:
Can you please ```console.log(todoTitle)``` here as per the instruction ``` Log the value of todoTitle in the console```

Reply:
I will add console.log(todoTitle) as per the instruction. Thank you so much.